back/spec/acceptance/ideas_update_spec.rb

```diff
@@ -379,14 +379,24 @@ def public_input_params(spec)
         let(:author) { input.author }
         let(:title_multiloc) { { 'en' => 'Changed title' } }
 
-        example_request 'Update an initiative' do
+        example 'Update an initiative' do
```

Comment:
Let's use consistent terminology - proposal everywhere and not initiative anymore.

Reply:
Well spotted, this was probably a result of copy-paste 😅